### DISCUSSION TITLE
The mean of data frame of UDP protocol

### DISCUSSION THREAD

**Discussion by Helene-Tatiana** [dannagle/PacketSender#362] [Q&A]
Hello everyone.
I'd just like please, to know when you put the destination address "YOU", who is that "You"? Is it my computer's IP address?

Regards!
![you UDP Protocol](https://github.com/dannagle/PacketSender/assets/115505718/ac214aa7-0289-47bf-9dbe-cbfe82b92dc5)

**Comment by dannagle** [comment]
The operating system does not always tell Packet Sender the true destination address, which is why it is always set to You if it received it.

The You could be any number of reasons. The most common reason is because it was sent to your computer's IP address (of which you may have several IP addresses, including 127.0.0.1). It could be because you are subscribed to a multicast address. It could be from a broadcast address. 

Also, seeing your screenshot made me realize that "You" would be better if it was also translated to other languages.

**Comment by Helene-Tatiana** [comment]
Ah okay.  I see better now. 
Thanks.
